main/ertp/ERTP-Guide/Overview.md

```diff
@@ -0,0 +1,258 @@
+- [javascript.info](https://javascript.info/async)
+- [Mozilla's Developer Docs](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous)
+
+**tyg todo: Could use beefing up about how and when Promises, async, and await are
+  actually used in contracts**
+
+## Object capabilities and ERTP
+
+ERTP implements [*object capabilities*](https://agoric.com/documentation/glossary/#object-capabilities). You can only use an object and issue commands to it if you have access to that object, not just its human-readable name or similar. For example, I might know (or make a good guess), that the mint that makes quatloos has the human-understandable alleged name of 'quatloos-mint'. But unless I have the actual object that is a `mint` object associated with the `quatloos` `brand` object, I can't use it to create a million `quatloos` and bet
+them all on Captain Kirk to win his gladiatorial match on Triskelion (see the [Wikipedia entry for the Star Trek episode](https://en.wikipedia.org/wiki/The_Gamesters_of_Triskelion)).
+
+## Creating and sending a fungible asset
+
+In ERTP, digital assets are created by a `mint`. You must have
+access to a `mint` object to create new assets of the `mint` object's
+associated `brand`. 
+
+For instance, let's say you want to create a new community
+currency called 'BaytownBucks'. You start by creating or 
+getting access to a `mint` object whose associated brand is 'BaytownBucks'.
+```js
+import { makeMint } from '@agoric/ertp';
+const baytownBucksMint = makeMint('BaytownBucks');
+```
+
+Now, use the `mint` to create 1000 new BaytownBucks.
+
+```js
+const { issuer, mint } = makeIssuerKit('BaytownBucks');
+const bb1000 = amountMath.make(1000);
+const newPayment = mint.mintPayment(bb1000);
+const bbpurse = issuer.makeEmptyPurse();
+const bb1000Amount = bbpurse.deposit(newPayment);
+```
+
+The act of minting created 1000 BaytownBucks and stored them together in a
+`payment`. We then deposited the `payment` into a `purse` for BaytownBucks.
+
+Now, let's distribute the BaytownBucks to others. To send assets
+in ERTP, we withdraw a `payment` from a `purse`.
+
+```js
+const paymentForAlice = purse.withdraw(bucks10);
+```
+
+Like our `purse`, this `payment` contains BaytownBucks, but unlike purses,
+we use payments to represent tokens in transit. A `payment` can be
+sent to someone else, a `purse` should never be sent to someone else.
+
+Now let's send the `payment` to Alice.
+
+Before she can receive a `payment` in BaytownBucks, Alice needs to 
+create an empty `purse` for BaytownBucks and otherwise get ready to
+receive payments. 
+
+For this to happen, Alice needs access to `baytownBucksIssuer()`.
+However, she does not need access to `baytownBucksMint()`.
+If she had access to the `mint()`, she could create BaytownBucks herself
+by calling `baytownBucksMint.mint()`. 
```

Comment:
This also needs to be updated for current ERTP. I think just checking against the current ERTP API docs should work.
```suggestion
by calling `baytownBucksMint.mintPayment()`. 
```